test/e2e/load_test.go

```diff
@@ -86,11 +86,7 @@ var _ = Describe("Podman load", func() {
 		result := podmanTest.Podman([]string{"load", "-q", "--signature-policy", "/etc/containers/policy.json", "-i", outfile})
 		result.WaitWithDefaultTimeout()
 		if IsRemote() {
-			Expect(result).To(ExitWithError())
-			Expect(result.ErrorToString()).To(ContainSubstring("unknown flag"))
-			result = podmanTest.Podman([]string{"load", "-i", outfile})
-			result.WaitWithDefaultTimeout()
-			Expect(result).Should(ExitCleanly())
```

Comment:
Judgment call: I could not figure out the point of this `load`. That is already tested elsewhere.